src/Testing/src/RetryAttribute.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel;
+
+namespace Microsoft.AspNetCore.Testing
+{
+    /// <summary>
+    /// Runs a test multiple times when it fails
+    /// This can be used on an assembly, class, or method name. Requires using the AspNetCore test framework.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Is this because this ships in a user visible package? Making it Never makes it really hard to type this in regular code and gives some people PTSD (@rynowak).

Reply:
I just copied https://github.com/dotnet/aspnetcore/blob/6cc7b9b9275dc0987f1c5eb385be5778f09d3717/src/Testing/src/RepeatAttribute.cs#L13 as a starting point so it wasn't specifically intentional by me